src/hooks.ts

```diff
@@ -27,4 +29,6 @@ export const defaultHooks: Hooks<EmissionMap, z.SomeZodObject> = {
   onAnyOutgoing: ({ event, logger, payload }) =>
     logger.debug(`Sending ${event}`, payload),
   onStartup: ({ logger }) => logger.debug("Ready"),
+  onError: ({ logger, event, error }) =>
+    logger.error(`${event} handling error`, error),
 };
```

Comment:
Would it make sense for the `Error` in the default `onError` hook to stringify the error?
```suggestion
  onError: ({ logger, event, error }) =>
    logger.error(`${event} handling error`, getMessageFromError(error)),
};
```

Reply:
Stringifying errors can reveal tracing, which I try to avoid for security reasons.
So I take only what is supposed to be string (the message), or I construct a string from certain props myself in that method, @feds01

Reply:
no, it's alright for the logger.